### DISCUSSION TITLE
Project Front Page with GitHub Pages?

### DISCUSSION THREAD

**Discussion by konn** [haskell/haskell-language-server#1319] [Ideas]
Currently, we have only GitHub repo for the project overview and documentation.
Since HLS has getting mature, it might be the time for us to consider setting up the overview/showcase and documentation pages, perhaps with GitHub Pages.

**Comment by berberman** [comment]
In addition, I think we can use CI to publish generated haddock to github pages after every push, which could help contributors, to some extent. Although there is documentation of the last release on hackage, it may be out-of-date compared to that of the master branch.

**Comment by Ailrun** [comment]
I'm not sure it is worth the effort for the building/maintaining one. It will be harder to update than simple `README.md`...

**Reply by Anrock** [reply]
If it's something simple and static, generated from markdown, than it should be comparable. Or we can just make a pretty page with screenshots, etc and link to often-updateable pieces of README from there.

**Reply by ndmitchell** [reply]
I agree that it seems like a moderate amount of work, for marginal benefit. People either way a Haskell IDE, or they don't - we aren't competing to be the "standard" Haskell IDE, since we already are. Given that, raw text with precise/concise information seems to beat nicely formatted pages.

**Reply by michaelpj** [reply]
Our README is quite long, and people do often not seem to read it all. _Possibly_ having a documentation site with better navigation might help.

**Reply by jneira** [reply]
Yeah i was about to add more demos to README but i was wondering if it would make it less readable (although you simply can scroll down until you dont see screen gifs or click in the index the section you are interested)

**Comment by jneira** [comment]
New issue about revisiting this: #2033
